pkg/logger/logger.go

```diff
@@ -5,20 +5,52 @@ import (
 	"strings"
 )
 
+var (
+	minGlobalLevel  = LevelDebug
+	minScopedLevels = make(map[string]Level)
+	registeredSinks []registeredSink
+)
+
+// SetLevel will elide all events (no matter if scoped or not) that has a
+// logging level lower than the provided value.
+//
+// If LevelSilence is used, then all logs will be disabled.
+func SetLevel(level Level) {
+	minGlobalLevel = level
+}
+
+// SetLevelScoped will elide all events for a given scope that has a logging
```

Comment:
nit: While I personally like the word elide, I think its usage is a bit too infrequent to expect a lot of people to understand it.
Suggest using a synonym that is more common, like:
```suggestion
// SetLevelScoped will suppress all events for a given scope that has a logging
```
find&replace in `logger.go` and `logger_test.go`:
  elide -> suppress
  Elided -> Suppressed

Reply:
Fare enough